packages/core/src/GenericBanner/GenericBanner.tsx

```diff
@@ -39,6 +40,47 @@ const CustomButton = styled(Button)`
   text-decoration: ${(props) => (props.buttonTextUnderline ? 'underline' : 'none')};
 `
 
+const BannerContent = styled(Box)`
```

Comment:
Flex maybe?

Reply:
Using Box here should be more correct.  Using Flex will make all the items occupy a whole line.  It depends on the design.

**Box**
<img width="1206" alt="image" src="https://github.com/priceline/design-system/assets/108819593/cd29b75f-4c56-44ba-9d9e-f099c2701b50">

**Flex**
<img width="1230" alt="image" src="https://github.com/priceline/design-system/assets/108819593/583d34fb-fc16-44e9-a438-0d9cb1f95a47">